### PR TITLE
Googlemapsios update

### PR DESCRIPTION
**Pull request by SotoiGhost**
* Updated to version 2.0.1
* Updated to version 2.1.0

### CODE REVIEW

File: Google.Maps/source/Google.Maps/Google.Maps.csproj [hunk 1]
```diff
@@ -38,6 +38,7 @@
   </ItemGroup>
   <ItemGroup>
     <Folder Include="Resources\" />
+    <Folder Include="Resources\" />
```
Comment (bholmes) [inline Google.Maps/source/Google.Maps/Google.Maps.csproj]: Extra Folder here.  Minor but lets fix it.

Comment (Redth) [inline Google.Maps/source/Google.Maps/Google.Maps.csproj]: Duplicate?

File: Google.Maps/samples/GoogleMapsAdvSample/GoogleMapsAdvSample/Info.plist [hunk 2]
```diff
@@ -9,13 +9,13 @@
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
 	<key>MinimumOSVersion</key>
-	<string>8.0</string>
+	<string>7.0</string>
```
Comment (bholmes) [inline Google.Maps/samples/GoogleMapsAdvSample/GoogleMapsAdvSample/Info.plist]: Is 7 now allowed?

File: Google.Maps/samples/GoogleMapsSample/GoogleMapsSample/Info.plist [hunk 3]
```diff
@@ -21,13 +21,15 @@
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
 	<key>MinimumOSVersion</key>
-	<string>8.0</string>
+	<string>7.0</string>
```
Comment (bholmes) [inline Google.Maps/samples/GoogleMapsSample/GoogleMapsSample/Info.plist]: Is 7 now allowed?

**Comment by bholmes** [comment]
@SotoiGhost you will need to add a change like the following commit to this PR.


https://github.com/xamarin/XamarinComponents/commit/a6620bdab1829891ad95ae07350f67b0f9648c59#diff-0b83f9dedf40d7356e5ca147a077acb4

Otherwise CI will fail.

**Comment by bholmes** [comment]
@SotoiGhost you need to add more to that last commit.  See the version change in cake.packages.config.  Also if there is a .ps file update it as well.

**Pull request merged**